### ISSUE TITLE
Checkout v2: Tab navigation working poorly

### ISSUE THREAD

**Issue by DavidRothstein** [Automattic/wp-calypso#89134]
### Quick summary

Tab navigation in checkout v2 has several issues that make it difficult for a keyboard user to tab through the page.

### Steps to reproduce

1. On an existing site, put something in your shopping cart.
2. Put `?checkoutVersion=2` into the URL.
3. Start tabbing through the document using your keyboard.

### What you expected to happen

Focus to proceed logically through the page and for it to be clear which item is focused.

### What actually happened

There were several problems:
1. Focus goes to the sidebar first (before the main content).  Not a disaster and not a new issue (checkout v1 has it too) but in v2 there's much more in the sidebar so it becomes more noticeable.
2. In the sidebar, focus goes immediately into the site preview iframe, meaning you can start tabbing through and navigating all the links on your actual site (within that iframe)!  This is not desirable, and also very confusing because it's extremely difficult to tell that focus is inside there, and it typically takes hitting the Tab button many times to get out.  Slack discussion suggested that this occurs on other site previews throughout Calypso also. These kind of site preview iframes should probably never get focus while tabbing. Addressed by https://github.com/Automattic/wp-calypso/pull/89136
3. Once you exit the site preview, the next couple elements (the billing term selector and the coupon link) are focusable, but there is no visual indicator that they are in focus so it's extremely difficult to know that's what's happening.

### Impact

Some (< 50%)

### Available workarounds?

Yes, difficult to implement

**Comment by jjchrisdiehl** [comment]
This should now be fixed 👍
